result of comparison of constant -1 with expression of type 'char' is always false

```
[ 16% 28/173] /usr/bin/c++ -DFMT_HEADER_ONLY=1 -I/wrkdirs/usr/ports/science/arbor/work/arbor-0.8/modcc -I/wrkdirs/usr/ports/science/arbor/work/.build/modcc/include -I/wrkdirs/usr/ports/science/arbor/work/arbor-0.8/arbor/include -I/wrkdirs/usr/ports/science/arbor/work/.build/arbor/include -I/wrkdirs/usr/ports/science/arbor/work/arbor-0.8/ext/tinyopt/include -isystem /usr/local/include -O2 -pipe -fstack-protector-strong -fno-strict-aliasing -O2 -pipe -fstack-protector-strong -fno-strict-aliasing   -DNDEBUG -fPIE -Wall -Wno-missing-braces -Wno-potentially-evaluated-expression -Wno-unused-function -Wno-unused-local-typedef -Wno-format-security -std=c++17 -MD -MT modcc/CMakeFiles/modcc.dir/modcc.cpp.o -MF modcc/CMakeFiles/modcc.dir/modcc.cpp.o.d -o modcc/CMakeFiles/modcc.dir/modcc.cpp.o -c /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/modcc/modcc.cpp
In file included from /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/modcc/modcc.cpp:8:
/wrkdirs/usr/ports/science/arbor/work/arbor-0.8/ext/tinyopt/include/tinyopt/tinyopt.h:894:18: warning: result of comparison of constant -1 with expression of type 'char' is always false [-Wtautological-constant-out-of-range-compare]
            if (c==EOF) break; 
                ~^ ~~~
1 warning generated.
```

Version: 0.8
clang-14
FreeBSD 13.1

--- a/ext/tinyopt/include/tinyopt/tinyopt.h
+++ b/ext/tinyopt/include/tinyopt/tinyopt.h
@@ -892,7 +892,7 @@ struct saved_options: private std::vector<std::string> {
         bool quote = false; // true => within single quotes.
         bool escape = false; // true => previous character was backslash.
         while (in) {
-            char c = in.get();
+            int c = in.get();
             if (c==EOF) break;
 
             if (quote) {

